api/src/middleware/__tests__/assessmentsRouter.ts

```diff
@@ -1,18 +1,242 @@
+              programAssessmentId
+            );
+            expect(mockFindRoleInProgram).toHaveBeenCalledWith(
+              programId,
+              enrolledParticipantPrincipalId
+            );
+            //get a list of program assessments for each program the user is enrolled in) to respond with a list of program assessments
+            expect(mockGetAssessmentsForProgram).toHaveBeenCalledWith(
+              programId
+            );
+            // get a list of curriculum assessments that correspond to each program assessment
+            expect(mockGetCurriculumAssessmentById).toHaveBeenCalledWith(
+              exampleAssessmentId,
+              false,
+              false
+            );
+            // call a (mock) function that gets the participant assessment summary for each program assessment where the user is a participant of that program
+            expect(mockGetAssessmentSubmissionsSummary).toHaveBeenCalledWith(
+              programAssessmentId,
+              enrolledParticipantPrincipalId
+            );
+            done(err);
+          }
+        );
+    });
+
+    it('should respond with a list of all assessments (without questions) for facilitator of one program', done => {
+      mockPrincipalId(facilitatorPrincipalId);
+      const facilitatorAssessmentListResponse = [
+        {
+          curriculum_assessment: curriculumAssessment,
+          program_assessment: programAssessment,
+          submissions_summary: facilitatorAssessmentSubmissionsSummary,
+        },
+      ];
+      // mock response from (function that gets a list of programs the user is enrolled in) to include one program
+      mockPrincipalEnrolledPrograms.mockResolvedValue([4]);
+      // mock response from (call a function that returns the permission of the user for each program (participant/facilitator)) to respond with facilitator for that one program
+      mockGetProgramIdByProgramAssessmentId.mockResolvedValue([2]);
+      mockFindRoleInProgram.mockResolvedValue({ title: 'facilitator' });
+      // mock response from (get a list of program assessments for each program the user is enrolled in) to respond with a list of program assessments
+      mockGetAssessmentsForProgram.mockResolvedValue(programAssessment);
+      // mock responses from (get a list of curriculum assessments that correspond to each program assessment) to respond with the corresponding curriculum assessment for each program assessment in that previous list
+      mockGetCurriculumAssessmentById.mockResolvedValue(curriculumAssessment);
+      // mock responses from (call a (mock) function that gets the facilitator assessment summary for each program assessment where the user is a facilitator of that program) to respond with the assessment summary for all participants for each program assessment
+      mockGetFacilitatorAssessmentSubmissionsSummary.mockResolvedValue(
+        facilitatorAssessmentSubmissionsSummary
```

Comment:
We'll want to take out the commented code before merging I think

Reply:
Thanks, I will.